src/_P016_IR.ino

```diff
@@ -14,23 +15,13 @@
 #include <IRrecv.h>
 #include <IRutils.h>
 
-#ifdef P016_Extended_Decoding // The following are only needed for extended decoding of A/C Messages
-#include <ir_Coolix.h>
-#include <ir_Daikin.h>
-#include <ir_Fujitsu.h>
-#include <ir_Gree.h>
-#include <ir_Haier.h>
-#include <ir_Hitachi.h>
-#include <ir_Kelvinator.h>
-#include <ir_Midea.h>
-#include <ir_Mitsubishi.h>
-#include <ir_Panasonic.h>
-#include <ir_Samsung.h>
-#include <ir_Tcl.h>
-#include <ir_Teco.h>
-#include <ir_Toshiba.h>
-#include <ir_Vestel.h>
-#include <ir_Whirlpool.h>
+#ifdef P035_Standardized_AC_Commands //They use the same library, so enable it for both plugins if already enabled in the other.
```

Comment:
Why have `P035_....` checks here in the P016 file?
If there is some relation between both, we should have these defines in the file where all build dependent defines are managed. ( `define_plugin_sets.h` )